app/phishing.html

```diff
@@ -1,67 +1,147 @@
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="./index-rtl.css"
+      title="rtl"
+      disabled
+    />
     <style>
-      *                          { margin: 0; padding: 0; box-sizing: border-box; }
-      body, html                 { background-color: rgb(217, 88, 70);
-                                   display: flex; flex-direction: column;
-                                   justify-content: center; align-items: center;
-                                   font-family: Roboto, Arial, sans-serif;
-                                   width: 100vw; min-height: 100vh; }
-      .content                   { display: flex; flex-direction: column; align-items: center;
-                                   width: 80%;
-                                   background-color: white; box-shadow: 0 0 15px #737373; }
-      .content__header           { display: flex; flex-direction: column; align-items: center; justify-content: center;
-                                   width: 100%;
-                                   color: rgb(217, 88, 70); border-bottom: 1px solid rgb(212, 212, 212);
-                                   padding: 2em; }
-      .content__header h1        { font-size: 24px; font-weight: normal; }
-      .content__header h1 i      { margin-right: 0.25em; }
-      .content__header img       { margin-bottom: 3em; width: 160px; }
-      .content__body             { background-color: rgb(245, 245, 245); font-size: 12pt; }
-      .content__body p           { margin: 2em; }
-      .content__body p a         { text-decoration: underline; color: inherit; cursor: pointer; }
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      body,
+      html {
+        background-color: var(--color-secondary-alternative);
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-family: Roboto, Arial, sans-serif;
+        width: 100vw;
+        min-height: 100vh;
+      }
+      .content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 80%;
+        background-color: var(--color-background-default);
+        box-shadow: 0 0 15px #737373;
+      }
+      .content__header {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        width: 100%;
+        color: var(--color-error-default);
+        border-bottom: 1px solid var(--color-border-default);
+        padding: 2em;
+      }
+      .content__header h1 {
+        font-size: 24px;
+        font-weight: normal;
+      }
+      .content__header h1 i {
+        margin-right: 0.25em;
+      }
+      .content__header img {
+        margin-bottom: 3em;
+        width: 160px;
+      }
+      .content__body {
+        background-color: var(--color-background-alternative);
+        font-size: 12pt;
```

Comment:
Not sure if it's need or will be covered in our main stylesheet but to be safe add 

```suggestion
        font-size: 12pt;
        color: var(--color-text-default);
```